src/lib/components/shared/vercel_hover.svelte

```diff
@@ -7,7 +7,7 @@
     let hover_glider_element: HTMLElement,
         glider_container_element: HTMLElement,
         GLIDER_TRANSITION_DURATION = 200,
-        is_hovered = false,
+        is_hovered = false, // Boolean switch flag
```

Comment:
Why this comment?
`// Boolean switch flag`

Reply:
ignore lol